<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListPublicationsDialog</class>
 <widget class="QDialog" name="ListPublicationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="KLineEdit" name="filterLineEdit"/>
     </item>
     <item>
      <widget class="KComboBox" name="filterComboBox"/>
     </item>
     <item>
      <widget class="KComboBox" name="libraryComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="createNew">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ListPublicationsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ListPublicationsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ListPublicationsDialog</receiver>
   <slot>applyFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>libraryComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ListPublicationsDialog</receiver>
   <slot>changeLibrary()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>applyFilter()</slot>
  <slot>changeLibrary()</slot>
 </slots>
</ui>
